koch.nim

```diff
@@ -199,13 +200,12 @@ proc buildTool(toolname, args: string) =
   nimexec("cc $# $#" % [args, toolname])
   copyFile(dest="bin" / splitFile(toolname).name.exe, source=toolname.exe)
 
-proc buildTools(latest: bool) =
+proc buildTools() =
   bundleNimsuggest()
   nimCompile("tools/nimgrep.nim", options = "-d:release")
   when defined(windows): buildVccTool()
   nimCompile("nimpretty/nimpretty.nim", options = "-d:release")
-  buildNimble(latest)
```

Comment:
Why exclude ``nimble`` here?

Reply:
because I'm already calling `kochExec "nimble"` immediately after `./koch boot` in runCI , which is the earliest point where nimble can be built; I'm building nimble as early as possible to enable its use in later stages (right now not needed, but may be in near future as part of CI/testing/etc)

note that `./koch tools` still builds nimble so nothing changed in that regard